modules/test/company-generator/src/main/java/com/liferay/company/generator/CompanyGenerator.java

```diff
@@ -89,6 +90,21 @@ protected void deactivate() {
 	private static final Log _log = LogFactoryUtil.getLog(
 		CompanyGenerator.class);
 
+	private static final ServiceTracker<ServletContext, ServletContext>
```

Comment:
@kyle-miho you are not doing this right, see https://github.com/liferay/liferay-portal/blob/master/modules/apps/application-list/application-list-api/src/main/java/com/liferay/application/list/adapter/PortletPanelAppAdapterRegistry.java#L79

Fix it, if still fails for you, include an error stack, and let's talk tomorrow.